### PR TITLE
Support for async parsers.

### PR DESCRIPTION
**Pull request by denismars**
Support for async parsers as current version was breaking for less scripts when in sync mode.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  this.cacheable && this.cacheable()
-  var data
-  compiler.compile(content, this.resourcePath, function (err, res) {
-    if (err) throw err
-    data = res
-  })
-  return data
-}
\ No newline at end of file
+module.exports = function(content) {
```
Comment (yyx990803) [inline index.js]: 2 space indent please :)

File: index.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+    this.cacheable && this.cacheable()
+    var cb = this.async()
+    compiler.compile(content, this.resourcePath, function(err, result) {
+        if(err) return cb(err)
+        cb(null, result)
+    });
```
Comment (yyx990803) [inline index.js]: remove semicolon here

**Comment by yyx990803** [comment]
Thanks! Looks good, just a few style issues.

File: index.js (hunk 2 above)
Comment (denismars) [inline index.js]: apologies for styling - will make sure I follow your guide for next time.

**Pull request merged**